### ISSUE TITLE
Migrate data from openerp 6.1 to openerp 8 

### ISSUE THREAD

**Issue by emnar** [OCA/OpenUpgrade#105]
Hi All,

I'm new to Openerp ( a junior Odoo developer) and  I need   to  migrate
data from openerp 6.1 to openerp 8 using XML-RPC, I connect to openerp
6.1 using a vpn server and to openerp 8 from my localhost.

I would like to know the steps to do that and  I also need an example of
migration data using XML-RPC.

Can you help me pleaaaaaaaaaaaaaaase !!!!

Thank you,

Emnar

**Comment by rvalyi** [comment]
@emnar 
that process will involve several steps, but sometimes a picture is worth thousands words ![llearning curve](http://www.sowdermd.com/wp-content/uploads/blog-learning-curve.jpg)

that may sound rude at first, but with the time, you'll probably understand why I advised. Good luck though...

tl;dr
forget trying that yourself

**Comment by StefanRijnhart** [comment]
Hi @emnar, your question is out of scope for this project, which aims at a more holistic upgrade method than just copying over the data. Nevertheless, as a hint: look at Erppeek. See http://erppeek.readthedocs.org/en/latest/. If you have any questions on Erppeek, post them here: https://www.odoo.com/forum/help-1

lol @rvalyi btw....
